nasa/harmony
# Harmony 1378 online documentation

## Jira Issue ID
HARMONY-1378

## Description
Adds a documentation page

## Local Test Steps
1. Check out this branch
2. run harmony locally
3. go to the landing page
4. click on the 'Documentation' link
5. Read the documentation
6. Verify that all the harmony links are to localhost (should use deployment environment)
7. Verify that all EDL links are to UAT (will be prod EDL in Harmony prod)
8. Bear in mind that this is not intended to be complete just yet. we can add things as we need them

## PR Acceptance Checklist
